pkg/cloud/baremetal/actuators/machine/actuator.go

```diff
@@ -46,14 +46,19 @@ const (
 	ProviderName = "baremetal"
 	// HostAnnotation is the key for an annotation that should go on a Machine to
 	// reference what BareMetalHost it corresponds to.
-	HostAnnotation                  = "metal3.io/BareMetalHost"
-	requeueAfter                    = time.Second * 30
-	externalRemediationAnnotation   = "host.metal3.io/external-remediation"
-	poweredOffForRemediation        = "remediation.metal3.io/powered-off-for-remediation"
-	requestPowerOffAnnotation       = "reboot.metal3.io/capbm-requested-power-off"
-	nodeLabelsBackupAnnotation      = "remediation.metal3.io/node-labels-backup"
-	nodeAnnotationsBackupAnnotation = "remediation.metal3.io/node-annotations-backup"
-	nodeFinalizer                   = "metal3.io/capbm"
+	HostAnnotation                    = "metal3.io/BareMetalHost"
+	requeueAfter                      = time.Second * 30
+	externalRemediationAnnotation     = "host.metal3.io/external-remediation"
+	poweredOffForRemediation          = "remediation.metal3.io/powered-off-for-remediation"
+	requestPowerOffAnnotation         = "reboot.metal3.io/capbm-requested-power-off"
+	nodeLabelsBackupAnnotation        = "remediation.metal3.io/node-labels-backup"
+	nodeAnnotationsBackupAnnotation   = "remediation.metal3.io/node-annotations-backup"
+	nodeFinalizer                     = "metal3.io/capbm"
+	machineRoleLabel                  = "machine.openshift.io/cluster-api-machine-role"
+	machineRoleMaster                 = "master"
+	annotationTimestampFormat         = time.RFC3339
+	powerOnDefaultTimeout             = 20 * time.Minute
```

Comment:
How is this timeout value used?

Reply:
It is used in case we were unable to identify/get MHC responsible for a machine.

Reply:
Could the name reflect that use somehow? `remediationPowerOnTimeout` maybe?